metadb/scrapers/musicbrainz.py

```diff
@@ -0,0 +1,43 @@
+import getpass as gt
+import sys
+import musicbrainzngs as mb
+
+#username = raw_input('Enter MusicBrainz Username: ')
+#password = gt.getpass('Password for {}: '.format(username))
+#mb.auth(username,password)
+
+mb.set_useragent(
+    "python-metadb-musicbrainz-scrape",
+    "0.1",
+    "https://github.com/MTG/metadb")
+
+
+
+def scrape(mbid):
+    try:
+        result = mb.get_recording_by_id(mbid, includes=["artists", "releases", "tags"])
+	
+	recordingName = result["recording"]["title"] #recording title
+	
+	artist = list()
+	artist_mbid = list()
```

Comment:
it's not a good idea to keep related information in two different objects, because you have to make sure that you keep them synchronised whenever you access them.
A better way would be to have a list for `artists`, and inside it have a dictionary containing both the id and the name. for example:
```python
artists.append({"id": item["artist"]["id"], "name": item["artist"]["name"]})
```